### PR TITLE
fix(common): improve LOGGING setting

### PR DESCRIPTION
**Pull request by obdulia-losantos**
This will reject blank values `""` too

### CODE REVIEW

**Comment by admbtlr** [comment]
We're in danger of scattering these checks everywhere...

File: aether-common-module/aether/common/conf/settings.py [hunk 1]
```diff
@@ -246,12 +303,21 @@
 if DJANGO_STORAGE_BACKEND == 'filesystem':
     DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
     HOSTNAME = os.environ['HOSTNAME']
+    if not HOSTNAME:
+        raise RuntimeError('Missing HOSTNAME')
```
Comment (MrPink) [inline aether-common-module/aether/common/conf/settings.py]: can you add `environment variable` after `HOSTNAME` to make it explicit, same with the others

Comment (obdulia-losantos) [reply aether-common-module/aether/common/conf/settings.py]: Done!

File: aether-common-module/aether/common/conf/settings.py [hunk 2]
```diff
@@ -246,12 +303,21 @@
...
     DEFAULT_FILE_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
     AWS_STORAGE_BUCKET_NAME = os.environ['BUCKET_NAME']
+    if not AWS_STORAGE_BUCKET_NAME:
+        logger.critical('Missing BUCKET_NAME environment variable!')
+
 elif DJANGO_STORAGE_BACKEND == 'gcs':
     DEFAULT_FILE_STORAGE = 'storages.backends.gcloud.GoogleCloudStorage'
     GS_BUCKET_NAME = os.environ['BUCKET_NAME']
+    if not GS_BUCKET_NAME:
+        logger.critical('Missing BUCKET_NAME environment variable!')
```
Comment (obdulia-losantos) [inline aether-common-module/aether/common/conf/settings.py]: @adamvert and @MrPink what do you prefer? To raise a `RuntimeError` or to send the critical log to the console/sentry?

Comment (admbtlr) [reply aether-common-module/aether/common/conf/settings.py]: I guess a `RuntimeError` is better, since we want it to die

Comment (quesebifurcan) [reply aether-common-module/aether/common/conf/settings.py]: Can we do both?

Comment (obdulia-losantos) [reply aether-common-module/aether/common/conf/settings.py]: Always!

**Review by admbtlr — APPROVED** [review]

**Pull request merged**